cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -110,6 +110,12 @@ while read line; do
   create-kubeconfig-secret "${token}" "${username}"
 done < /srv/kubernetes/known_tokens.csv
 
+# Create adminion_control objects first if defined before any other addon services
```

Comment:
Please add a comment to start_addon to the effect that if we change it to create addons in a namespace other than default we should still create the limits for the default namespace.